src/Raven.Server/Documents/BucketStats.cs

```diff
@@ -0,0 +1,17 @@
+using System.Runtime.InteropServices;
+
+namespace Raven.Server.Documents
+{
+    [StructLayout(LayoutKind.Explicit)]
```

Comment:
Probably we need to add Pack = 1 here? I see only `long` here, but still it would not hurt?

Reply:
done